src/uci.cpp

```diff
@@ -275,6 +275,10 @@ void UCI::loop(int argc, char* argv[]) {
       else if (token == "d")        sync_cout << pos << sync_endl;
       else if (token == "eval")     trace_eval(pos);
       else if (token == "compiler") sync_cout << compiler_info() << sync_endl;
+      else if (token == "#") {
```

Comment:
What will happen with `#comment` (i.e. no space) ?

Reply:
Won't work.  I decided to do it that way, in case some future genius wants something like #include

Reply:
IMHO it's more natural to just check for '#'. Lots of config files on Linux do it this way (see e.g. https://salsa.debian.org/ssh-team/openssh/-/raw/master/sshd_config) and it's very convenient to quickly comment/uncomment a "true uci command".

Even then, #include and the likes are always possible by preprocessing those first.

Reply:
Fine with me.  I just want the functionality in there.  :)